src/display/api.js

```diff
@@ -244,6 +248,7 @@ function getDocument(src, pdfDataRangeTransport,
   }
 
   params.rangeChunkSize = params.rangeChunkSize || DEFAULT_RANGE_CHUNK_SIZE;
+  params.disableNativeImageDecoder = params.disableNativeImageDecoder === true;
```

Comment:
I suppose that we could even do e.g. ` ... = params.disableNativeImageDecoder === true || typeof document === 'undefinef';` if we wanted to provide a decent out-of-the-box behaviour for Node.js?